gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,8 +66,23 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
+      queue_failover_executed_event
+    end
+
     private
 
+    def queue_failover_executed_event
+      connection = pg_connection(@database_yml.pg_params_from_database_yml)
+      connection.exec(<<-SQL)
```

Comment:
This makes me nervous ... are we sure we don't want to make it a rake task @Fryguy @gtanzillo ?